std_msgs/Header      header
geometry_msgs/Pose[] poses
geometry_msgs/Pose[] beacons
geometry_msgs/Pose   predicted_pose